### PR TITLE
Add files with hero to master

### PR DESCRIPTION
**Pull request by Tarnoruda**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (krichert) [inline .vscode/settings.json]: what is it? add rule to .gitignore file

File: index.html [hunk 2]
```diff
@@ -1,19 +1,45 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <!-- Required meta tags -->
```
Comment (krichert) [inline index.html]: do we need this comment?

File: index.html [hunk 3]
```diff
@@ -1,19 +1,45 @@
...
 <body>
-    <header>Hero</header>
-    <nav>Nav Bar</nav>
-    <main>
-    <section>Function</section>
-    <section>Authors</section>
-    <section>Extra Functions</section>
-    </main>
-    <footer>Contact&info</footer>    
+        <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
```
Comment (krichert) [inline index.html]: example in id?

**Review by krichert — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Tarnoruda) [reply index.html]: removed

File: .vscode/settings.json (hunk 1 above)
Comment (Tarnoruda) [reply .vscode/settings.json]: resolved/deleted

File: .vscode/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+.vscode
+settings.json
```
Comment (krichert) [inline .vscode/.gitignore:2]: it is not need, you just need .vscode

File: index.html [hunk 5]
```diff
@@ -1,19 +1,44 @@
...
+                  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                  <span class="sr-only">Previous</span>
+                </a>
+                <a class="carousel-control-next" href="#carouselControls" role="button" data-slide="next">
+                  <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                  <span class="sr-only">Next</span>
+                </a>
+              </div>
+
+<!-- Optional JavaScript -->
```
Comment (krichert) [inline index.html:38]: is it optional for us?

**Review by krichert — APPROVED** [review]

File: .vscode/.gitignore (hunk 4 above)
Comment (Tarnoruda) [reply .vscode/.gitignore:2]: resolved

File: index.html (hunk 5 above)
Comment (Tarnoruda) [reply index.html:38]: removed comments

**Pull request merged**